lib/modules/datasource/sbt-package/index.ts

```diff
@@ -15,8 +16,8 @@ import {
   parseIndexDir,
 } from './util';
 
-export class SbtPackageDatasource extends Datasource {
-  static id = 'sbt-package';
+export class SbtPackageDatasource extends MavenDatasource {
```

Comment:
Does it really extend anything or instead reimplement everything?

Reply:
It's calling base for gitlab

https://github.com/renovatebot/renovate/blob/765a76319be09383f1998eb76745f358db867214/lib/modules/datasource/sbt-package/index.ts#L156-L160